### ISSUE TITLE
Having an Issue Building Neovim .7-dev on Fedora GNU/Linux

### ISSUE THREAD

**Issue by FOSSilizedDaemon** [neovim/neovim#17492]
### Neovim version (nvim -v)

NVIM v.0.7.0-dev

### Vim (not Nvim) behaves the same?

No idea as this is a build issue.

### Operating system/version

Fedora GNU/Linux v.5.16.9

### Terminal name/version

gnome-terminal and st have been tested, but again not really applicable.

### $TERM environment variable

xterm-256color

### Installation

source

### How to reproduce the issue

1. Download the source code (zip) of neovim 7 release.
2. Unzip it.
3. sudo make CMAKE_BUILD_TYPE=RelWithDebInfo
4. sudo make install
5. nvim

### Expected behavior

It reads my configuration fine as expected.

### Actual behavior

It says it cannot find `vim.uri`, exact errors below

Error detected while processing /home/user/document/development/miscellaneous/dotfiles/src/dotfiles/home/.config/nvim/init.lua:
E5113: Error while calling lua chunk: vim.lua:0: module 'vim.uri' not found:
        no field package.preload['vim.uri']
        no file './vim/uri.lua'
        no file '/home/user/.usr/src/source-code/userland/neovim/.deps/usr/share/luajit-2.1.0-beta3/vim/uri.lua'
        no file '/usr/local/share/lua/5.1/vim/uri.lua'
        no file '/usr/local/share/lua/5.1/vim/uri/init.lua'
        no file '/home/user/.usr/src/source-code/userland/neovim/.deps/usr/share/lua/5.1/vim/uri.lua'
        no file '/home/user/.usr/src/source-code/userland/neovim/.deps/usr/share/lua/5.1/vim/uri/init.lua'
        no file './vim/uri.so'
        no file '/usr/local/lib/lua/5.1/vim/uri.so'
        no file '/home/user/.usr/src/source-code/userland/neovim/.deps/usr/lib/lua/5.1/vim/uri.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
        no file './vim.so'
        no file '/usr/local/lib/lua/5.1/vim.so'
        no file '/home/user/.usr/src/source-code/userland/neovim/.deps/usr/lib/lua/5.1/vim.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
        [C]: in function 'require'
        vim.lua: in function '__index'
        /home/user/.config/nvim/lua/module/key-binding/visual.lua:9: in main chunk
        [C]: in function 'require'
        /home/user/.config/nvim/lua/key-binding.lua:10: in main chunk
        [C]: in function 'require'
        /home/user/.config/nvim/init.lua:12: in main chunk
E484: Can't open file /usr/local/share/nvim/syntax/syntax.vim
Press ENTER or type command to continue

**Comment by clason** [comment]
Make sure that the installed Nvim is actually first on your path and `nvim` isn't launching a system package binary with a different runtime (or vice versa).

**Comment by FOSSilizedDaemon** [comment]
> Make sure that the installed Nvim is actually first on your path and `nvim` isn't launching a system package binary with a different runtime (or vice versa).

I uninstalled the version of neovim that Fedora packages and the only one I have installed is this dev version. I mostly installed it as I am using neovim as my RSS feeder (nvim + sfeed) and in order to use a specific binding I needed I was told to upgrade. I made sure to uninstall the system binary and then clear all of the cache and data left from that install.
